---
title: "Introduction to DiscreteFDR"
#author: "Florian Junge, Christina Kihn"
#date: "`r Sys.Date()`"
output: 
  rmarkdown::html_vignette:
    df_print: kable
    toc: true
vignette: >
  %\VignetteIndexEntry{Introduction to DiscreteFDR}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  echo = TRUE,
  message = FALSE,
  eval = TRUE,
  comment = "#>",
  fig = TRUE,
  fig.width = 7,
  fig.height = 5,
  fig.align = 'center'
)
```

Multiple testing procedures are important tools for identifying statistically significant findings in massive and complex data while controlling a specific error rate. An important focus has been given to methods controlling the false discovery rate (FDR), i.e., the expected proportion of falsely rejected hypotheses among all rejected hypotheses, which has become the standard error rate for high dimensional data analysis. Since the original procedure of [Benjamini and Hochberg (1995)](https://doi.org/10.1111/j.2517-6161.1995.tb02031.x), much effort has been undertaken to design FDR controlling procedures that adapt to various underlying structures of the data, such as the quantity of signal, the signal strength and the dependencies, among others.

The **R** package **DiscreteFDR**, presented in this paper, deals with adaptation  to discrete and non-identically distributed test statics by implementing procedures developed by [Döhler, Durand and Roquain (2018)](https://doi.org/10.1214/18-EJS1441) (in the sequel abbreviated as [[DDR]](https://doi.org/10.1214/18-EJS1441)). This type of data arises in many relevant applications, in particular when data represent frequencies or counts. Examples can be found in  clinical studies (see e.g., [Westfall and Wolfinger (1997)](https://doi.org/10.1080/00031305.1997.10473577)), genome-wide association studies (GWAS) (see e.g., [Dickhaus, Straßburger et al. (2012)](https://www.degruyter.com/view/j/sagmb.2012.11.issue-4/1544-6115.1776/1544-6115.1776.xml)) and next generation sequencing data (NGS) (see e.g., [Doerge and Chen (2015)](https://arxiv.org/abs/1502.00973)}).
  
***  
  
```{r child = "toyexample.Rmd"}
```

```{r child = "furtheranalyses.Rmd"}
```
